bin/test-example

```diff
@@ -50,7 +58,9 @@ if [ "$exampletype" = "success" ]; then
     # Implicit exit value: this is last thing in this path,
     # so the script's exit value = stack's exit value.
     # If statements are reordered, please preserve this property.
-    runstack "test"
+    if ! runstack "test"; then
+        exit 1
+    fi
```

Comment:
Well, it has preserved the intent, but in that case the three lines above **need to be removed** since now it's no longer an implicit exit value.